src/app/shared/services/auth/auth.service.ts

```diff
@@ -68,7 +71,10 @@ export class AuthService {
 			// Set user data in localStorage.
 			this.setUserData(result.user);
 
-			if (result.user?.email?.endsWith('matei@07internet.ro')) {
+			if (
+				result.user?.email?.endsWith('matei@07internet.ro') ||
```

Comment:
Perhaps create a table for the users with permissions rather than hardcoding the admins? Is that planned?

Reply:
Yes, I'll plan to do a role-based user permissions soon.